src/components/about/SilceZone.tsx

```diff
@@ -10,12 +12,27 @@ interface Props {
 }
 
 const SliceZone = ({ slice }: Props) => {
+  const formatCardItems = (sliceItems: CardListItemCMS[]) => {
+    return sliceItems.map((item) => {
+      return {
+        title: item.title,
+        content: <RichText render={item.card_content} />,
+      }
+    })
+  }
+
```

Comment:
Tu pode tirar esses return já que os `=>` são return
```suggestion
  const formatCardItems = (sliceItems: CardListItemCMS[]) =>
   sliceItems.map((item) =>  ({
        title: item.title,
        content: <RichText render={item.card_content} />,
      })
    )
```